src/ray/raylet/worker_pool_test.cc

```diff
@@ -179,16 +195,22 @@ TEST_F(WorkerPoolTest, StartupWorkerCount) {
       } else {
         ASSERT_TRUE(worker_pool_.NumWorkerProcessesStarting() ==
                     expected_worker_process_count);
-        ASSERT_TRUE(static_cast<int>(i * languages.size() + j) >=
+        ASSERT_TRUE(static_cast<int>(i * LANGUAGES.size() + j) >=
                     expected_worker_process_count);
       }
     }
   }
-  // Check number of starting worker processes
+  // Check number of starting workers
   ASSERT_EQ(worker_pool_.NumWorkerProcessesStarting(), expected_worker_process_count);
-  ASSERT_TRUE(worker_pool_.NumWorkerProcessesStarting() <
-              static_cast<int>(desired_initial_worker_process_count_per_language *
-                               languages.size()));
+}
+
+TEST_F(WorkerPoolTest, InitialWorkerProcessCount) {
+  worker_pool_.Start(1);
+  // Because one worker process has exactly NUM_WORKERS_PER_PROCESS workers,
+  // The worker_pool.NumWorkersStarting() may not equal to num_workers.
```

Comment:
This comment doesn't seem to belong here?

Reply:
Maybe the comment isn't clear. How about this?

> Here we try to start only 1 worker for each worker language. But since each worker process contains exactly NUM_WORKERS_PER_PROCESS (3) workers here, it's expected to see 3 workers for each worker language, instead of 1.

If this is better, I'll update the comment and add one more assertion `ASSERT_NE(worker_pool_.NumWorkersStarting(), 1 * LANGUAGES.size())`.

Reply:
I see, I misunderstood the code. This new comment is clearer.

Reply:
Done.